packages/vue/src/components/layouts/MainLayout.vue

```
@@ -7,7 +7,8 @@
 <script setup lang="ts">
 import { useI18n } from '@/composables';
 
-const { lang } = useI18n('home');
+const { lang, i18nApp } = useI18n('home');
+console.log('📟 - i18nApp → ', i18nApp.language);
```

ESLint

[error] Unexpected console statement.

(no-console)